MekHQ/data/scenariomodifiers/OverwhelmingEnemyGroundReinforcements.xml

```diff
@@ -0,0 +1,44 @@
+<AtBScenarioModifier>
+	<additionalBriefingText>Overwhelming enemy ground reinforcements have been detected approaching the area.</additionalBriefingText>
+	<allowedMapLocations>
+		<allowedMapLocation>AllGroundTerrain</allowedMapLocation>
+		<allowedMapLocation>SpecificGroundTerrain</allowedMapLocation>
+	</allowedMapLocations>
+	<benefitsPlayer>false</benefitsPlayer>
+	<eventTiming>PostForceGeneration</eventTiming>
+	<forceDefinition>
+		<actualDeploymentZone>-1</actualDeploymentZone>
+		<allowAeroBombs>true</allowAeroBombs>
+		<allowedUnitType>-2</allowedUnitType>
+		<arrivalTurn>-3</arrivalTurn>
+		<canReinforceLinked>true</canReinforceLinked>
+		<contributesToBV>false</contributesToBV>
+		<contributesToMapSize>false</contributesToMapSize>
+		<contributesToUnitCount>false</contributesToUnitCount>
+		<deploymentZones>
+			<deploymentZone>1</deploymentZone>
+			<deploymentZone>2</deploymentZone>
+			<deploymentZone>3</deploymentZone>
+			<deploymentZone>4</deploymentZone>
+			<deploymentZone>5</deploymentZone>
+			<deploymentZone>6</deploymentZone>
+			<deploymentZone>7</deploymentZone>
+			<deploymentZone>8</deploymentZone>
+			<deploymentZone>9</deploymentZone>
+		</deploymentZones>
+		<destinationZone>4</destinationZone>
+		<fixedUnitCount>-1</fixedUnitCount>
+		<forceAlignment>2</forceAlignment>
+		<forceMultiplier>1.0</forceMultiplier>
+		<forceName>Opfor Ground Reinforcements</forceName>
```

Comment:
Not really. "Primary Opfor" is keyed in as an explicit force name reference in a lot of modifiers, and I don't really remember what references what any more, so that means I'd have to re-test all these definitions.